packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts

```diff
@@ -87,9 +87,10 @@ describe('channel results', () => {
     const createChannel = createChannelFromState(signedStates[0]);
     await wallet.pushMessage({objectives: [createChannel]});
 
-    await expectResults(wallet.pushMessage({signedStates: signedStates.map(serializeState)}), [
-      {turnNum: zero, status: 'proposed'},
-    ]);
+    await expectResults(
+      wallet.pushMessage({signedStates: signedStates.map(ss => serializeState(ss))}),
```

Comment:
Was this change deliberate, or is this prettier?

Reply:
I think deliberate, but I can't remember exactly why I did it